deploy/scripts/src/mobile/backend-e2e.test.ts

```diff
@@ -0,0 +1,60 @@
+import { type Options } from 'k6/options'
+import { describeProfile, type ProfileList, selectProfile } from '../common/utils/config/load-profiles'
+import {
+  postVerifyAuthorizeRequest,
+  postResourceOwnerDocumentGroups,
+  getBiometricTokenV2,
+  postFinishBiometricSession,
+  getRedirect,
+  postToken,
+  postUserInfoV2
+} from './utils/backend'
+import { sleep } from 'k6'
+
+const profiles: ProfileList = {
+  smoke: {
+    backendJourney: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1, // start with one iteration
+      timeUnit: '1s',
+      preAllocatedVUs: 50, // Calculation: 5 journeys / second * 10 seconds average journey time
+      maxVUs: 70, // Calculation: 5 journeys / second * 14 seconds maximum journey time
+      stages: [
+        { target: 5, duration: '30s' }, // linear increase from 1 iteration per second to 5 iterations per second for 30 seconds
+        { target: 5, duration: '30s' } // maintain 5 iterations per second for 30 seconds
+      ],
+      exec: 'backendJourney'
+    }
+  }
+}
+
+const loadProfile = selectProfile(profiles)
+
+export const options: Options = {
+  // httpDebug: 'full',
```

Comment:
do we need this comment?

Reply:
Again, I think this was left over from the FE config we copied across. With this option enabled, it logs the entire requests and responses in full, but is this something that would be useful for us in debugging? It logs a lot of information, even the HTML returned from the GET requests, so thought it was maybe providing too much detail

@raluca-alecu Tagging you in case you have any further insight on the usefulness of this option!

Reply:
None of the other performance test scripts elsewhere in the repo have this option enabled. I'd be happy to omit, given we generally don't leave comments unless they're necessary to explain something

Reply:
I think the logs are also a bit hard to digest at the level of detail that they provide, if you have lots of iterations running at once I feel it wouldn't be easy to sift through them (they also get truncated by my IDE terminal, there are that many of them 😅 )

Reply:
Yep we definitely don't want this option enabled when running a load test - it will slow the test tool down. 

I'll leave it up to you whether to leave it commented out or remove from the file entirely

Reply:
I think we can remove the comment 👍

Reply:
Removed!